types.go

```diff
@@ -248,7 +248,7 @@ type RootFS struct {
 
 type CPUInfo struct {
 	UserHz  int `json:"user_hz"`
-	MHZ     int
+	MHZ     interface{}
```

Comment:
instead of interface{} make it a StringOrInt https://github.com/luthermonson/go-proxmox/blob/main/types.go#L773 which will make it backwards compatible